py-functions-amazon/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
Please, for all the scripts: Add a comment with a short description of what it is doing and a brief explanation how to use it (e.g. with an example).